virttest/utils_test/__init__.py

```diff
@@ -240,10 +240,10 @@ def get_time(session, time_command, time_filter_re, time_format):
                 msg = "Fail to get guest time offset. Command "
                 msg += "'%s', output: %s" % (time_command, output)
                 raise exceptions.TestError(msg)
-            if re.match('PM', guest_time):
+            if re.search('PM', guest_time):
                 hour = re.findall('\d+ (\d+):', guest_time)[0]
                 hour = str(int(hour) + 12)
-                guest_time = re.sub('\d+\s\d+:', "\d+\s%s:" % hour,
+                guest_time = re.sub('\d+/\s\d+:', "\d+\s%s:" % hour,
```

Comment:
@luckyh 
In my test, the previous pattern can not  get right result:
>>> guest_time="3/30/2017 3:28:47 PM"
>>> if re.search('PM',guest_time):
...     hour = re.findall('\d+ (\d+):', guest_time)[0]
...     hour = str(int(hour) + 12)
...     guest_time = re.sub('\d+\s\d+:', "\d+\s%s:" % hour,guest_time)[:-3]
...     print guest_time
... 
3/30/\d+\s15:28:47

And when use the new pattern,  it is ok.
...     guest_time = re.sub('\d+/\s\d+:', "\d+\s%s:" % hour,guest_time)[:-3]
...     print guest_time
... 
3/30/2017 3:28:47

Thanks~